src/grt/grt-avhpi.adb

```diff
@@ -27,6 +27,19 @@ with Grt.Vstrings; use Grt.Vstrings;
 with Grt.Rtis_Utils; use Grt.Rtis_Utils;
 with Grt.To_Strings;
 
+-- Acronyms that I don't understand:
+   -- Rti
```

Comment:
rti: run time information.  That's all the data emitted by translate to describe vhdl types and subtypes.